hugegraph-llm/src/tests/operators/index_op/test_semantic_id_query.py

```
@@ -0,0 +1,223 @@
+        # Clean up the temporary directory
+        shutil.rmtree(self.test_dir)
+
+    @patch("hugegraph_llm.operators.index_op.semantic_id_query.VectorIndex")
+    @patch("hugegraph_llm.operators.index_op.semantic_id_query.resource_path")
+    @patch("hugegraph_llm.operators.index_op.semantic_id_query.huge_settings")
+    @patch("hugegraph_llm.operators.index_op.semantic_id_query.PyHugeClient", new=MockPyHugeClient)
+    def test_init(self, mock_settings, mock_resource_path, mock_vector_index_class):
+        # Configure mocks
+        mock_settings.graph_name = "test_graph"
+        mock_settings.topk_per_keyword = 5
+        mock_resource_path = "/mock/path"
```

Ruff (0.11.9)

Local variable `mock_resource_path` is assigned to but never used

Remove assignment to unused variable `mock_resource_path`

(F841)